Detectors/ITSMFT/ITS/tracking/src/Tracker.cxx

```diff
@@ -41,6 +40,7 @@ Tracker::Tracker(o2::its::TrackerTraits* traits)
   mMemParams.resize(1);
   assert(traits != nullptr);
   mTraits = traits;
+  mMatLayerCylSet = o2::base::MatLayerCylSet::loadFromFile();
```

Comment:
Can you call this only if the parameter is set? Otherwise I get a seg fault if the matbud.root file is not in the directory.